Client/Frontend/Widgets/ThumbnailCell.swift

```diff
@@ -130,7 +197,9 @@ class ThumbnailCell: UICollectionViewCell {
     var imagePadding: CGFloat {
         didSet {
             imageView.snp_remakeConstraints({ make in
-                make.top.bottom.left.right.equalTo(self.imageWrapper).insets(UIEdgeInsetsMake(imagePadding, imagePadding, imagePadding, imagePadding))
+                let insets = UIEdgeInsetsMake(imagePadding, imagePadding, imagePadding, imagePadding)
+                make.top.left.right.equalTo(self.imageWrapper).insets(insets)
+                make.bottom.equalTo(textWrapper.snp_top).offset(-imagePadding) // .insets(insets)
             })
```

Comment:
Also super Nit: but could you move the var declarations up with the other var declarations?